feature_tests/src/structs.rs

```diff
@@ -4,9 +4,14 @@ pub mod ffi {
 
     #[diplomat::opaque]
     #[diplomat::transparent_convert]
+    #[diplomat::attr(kotlin, disable)]
     pub struct Opaque(String);
 
+    #[diplomat::opaque]
+    pub struct OtherOpaque(String);
```

Comment:
This is just temporary until I get support for other return types. Then I can just use the existing structs and not pollute this file.